templates/login.html

```diff
@@ -1,62 +1,61 @@
-{% extends 'layout.html' %}
+<!-- {% extends 'layout.html' %} -->
 {% block links %}
 <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
 {% endblock %}
+{%block scripts%}
+<script type ="text/javascript">
 
+function bookdisplay(ele){
+  console.log(ele)
```

Comment:
remove all the console.log statements as this might lead to unwanted information leakage.